.travis.yml

```diff
@@ -1,19 +1,36 @@
 language: go
 go: 1.6
-sudo: false
+sudo: required
```

Comment:
Is it tricky to get this to keep running in Travis' container-based infrastructure? Just asking, no need to change this if it's really hard or time consuming.